src/Validation/Validation.php

```diff
@@ -654,7 +654,9 @@ public static function multiple($check, array $options = [], $caseInsensitive =
         $defaults = ['in' => null, 'max' => null, 'min' => null];
         $options += $defaults;
 
-        $check = array_filter((array)$check);
+        $check = array_filter((array)$check, function ($value) {
```

Comment:
Nano-optimization probably to not use strlen(), I am fine changing if you prefer.